src/jiradeps/main.py

```diff
@@ -32,7 +32,7 @@
               default=os.path.join(os.getcwd(), CONFIG_FILENAME),
               help='Config file to be used '
                    f'(defaults to CWD/{CONFIG_FILENAME})')
-@click.option('--password', help='Jira password')
+@click.option('--password', '-p', help='Jira password')
```

Comment:
While I approve of the short option, it is quite surprising given the title of the PR.

Reply:
Well, it is tested with the integration test ;-)